ingestion/tests/integration/ometa/test_ometa_messaging_service_api.py

```diff
@@ -0,0 +1,216 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+OpenMetadata high-level API Messaging Service tests
+"""
+import uuid
+from unittest import TestCase
+
+from metadata.generated.schema.api.services.createMessagingService import (
+    CreateMessagingServiceRequest,
+)
+from metadata.generated.schema.entity.data.messaging import Messaging
```

Comment:
pytests are failing because module `metadata.generated.schema.entity.data.messaging` does not exist can you please remove this import